index.html

```diff
@@ -16,64 +16,107 @@
       <div class="main-card">
```

Comment:
For your `main-card` you could consider using a `section`-element as it makes more sense semantically.